### PR TITLE
Update platform deps to 7.1.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by mbarkley**

### CODE REVIEW

**Comment by mbarkley** [comment]
@porcelli Please merge this when you have time. I think this is blocking some of the Entando devs.

/cc @pefernan

File: appformer-deps/pom.xml [hunk 1]
```diff
@@ -75,7 +75,16 @@
...
+        <artifactId>kie-platform-bom</artifactId>
+        <type>pom</type>
+        <version>${version.org.kie}</version>
+        <scope>import</scope>
+      </dependency>
+
+      <!-- KIE 3rd party -->
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-parent</artifactId>
```
Comment (psiroky) [inline appformer-deps/pom.xml]: This is not needed since you already (transitively) inherit from kie-parent

**Comment by porcelli** [comment]
@mbarkley just waiting for the last change requested by @psiroky

**Comment by mbarkley** [comment]
@psiroky @porcelli Removed `kie-parent` from the dependency management and fixed a CDI dependency error that I accidentally introduced the first time around while fixing a compilation issue.

**Comment by porcelli** [comment]
thx @mbarkley - just waiting for the green status to merge :)

**Pull request merged**